skrub/dataframe/_polars.py

```diff
@@ -14,6 +16,54 @@
 from skrub._utils import atleast_1d_or_none
 
 
+def make_dataframe(X, index=None):
+    """Convert an dictionary of columns into a Polars dataframe.
+
+    Parameters
+    ----------
+    X : mapping from column name to 1d iterable
+        Input data to convert.
+
+    index : 1d array-like, default=None
+        Unused since polars doesn't use index.
+        Only here for compatibility with Pandas.
+
+    Returns
+    -------
+    X : Polars dataframe
+        Converted output.
+    """
+    if not isinstance(X, dict) or not all(
```

Comment:
would it make sense to raise an error when `index` is not `None`? having an index when we are building a polars dataframe might point to a bug. If not, maybe we could add a `del index` statement to make it clear we don't use it

Reply:
Yes, I think it's reasonable to raise an error when `index` is not `None` for Polars.